subsys/bluetooth/mesh/proxy_srv.c

```diff
@@ -832,8 +832,8 @@ static void gatt_connected(struct bt_conn *conn, uint8_t err)
 	struct bt_conn_info info;
 
 	bt_conn_get_info(conn, &info);
-	if (info.role != BT_CONN_ROLE_PERIPHERAL ||
-	    !service_registered) {
+	if (info.role != BT_CONN_ROLE_PERIPHERAL || !service_registered ||
+		info.id != BT_ID_DEFAULT) {
```

Comment:
This split line is not correctly aligned. It should start right after `if (` on the first line. You're doing it correctly in the two earlier cases, so just follow the same style here.